plugins/GSdx/GSTextureCache.cpp

```diff
@@ -48,7 +48,9 @@ GSTextureCache::GSTextureCache(GSRenderer* r)
 
 	m_paltex = theApp.GetConfigB("paltex");
 	m_can_convert_depth &= s_IS_OPENGL; // only supported by openGL so far
-	m_crc_hack_level = theApp.GetConfigI("crc_hack_level");
+	m_crc_hack_level = theApp.GetConfigT<CRCHackLevel>("crc_hack_level");
+	if (m_crc_hack_level == CRCHackLevel::Automatic)
+		m_crc_hack_level = AutoSelectCRCHackLevel(theApp.GetConfigT<GSRendererType>("Renderer"));
```

Comment:
IIRC depth emulation can malfunction if you set the renderer to OGL Software in the plugin settings window, then switch to OGL Hardware with the F9 key. This issue with `s_IS_OPENGL` might be at the source of it.